bark/runtime/viewer/viewer.py

```diff
@@ -406,7 +409,7 @@ def drawAgent(self, agent, color, alpha, facecolor, hatch=''):
                                                shape.rear_dist)) * math.sin(pose[2]) + pose[1]
 
             if self.draw_agent_id:
-                self.drawText(position=(centerx, centery), rotation=180.0*(1.0+pose[2]/math.pi), text="{}".format(agent.id),
+                self.drawText(position=(centerx, centery + 2.), rotation=180.0*(1.0+pose[2]/math.pi), text="{}".format(agent.id),
```

Comment:
adding + 2 to the center seems arbitrary, especially when the vehicles drive in two directions. The id's will then be displayed with a large offset.